cloudflare_worker/src/utils.rs

```diff
@@ -43,6 +43,6 @@ pub fn get_last_error_message() -> JsValue {
     }
 }
 
-pub fn anyhow_error_to_js_value(error: anyhow::Error) -> JsValue {
+pub fn anything_to_js_error<T: std::fmt::Debug>(error: T) -> JsValue {
```

Comment:
Maybe rename `to_js_error` in the style of methods like `to_string`.

Reply:
Done